Some changes to Mario's Firebrand

Nakedkoops opened the pull request:
**Changes:**

[=] Given a weaker lingering hitboxes that's been given the following changes:
    - Damage 8
    - KBG 100
[=] The lingering hitboxes last for 7 frames
[=] shoulder hitbox size has been reduced from 3.5 to 2.5 

[+] The arm hitbox has been shifted to the right (aka x value increased) from 5 to 5.5  

[-] Strong hitboxes last from 5 frames to 3 frames 
[-] arm hitbox has reduced size from 6.5 to 5.0
[-] strong shoulder hitbox is also slightly weaker to differentiate it from the strong arm hitbox, changes are:
    -Damage 10
    - KGB 110

Edit: Changed the Arm hitbox from 5.5, screenshots have been updated to more the more recent changes
Edit(2): Changed lingering hitbox frames from 7 -> 5. The entire move's frame data now has 8 attacking frames.
Edit (3): Changed some effects on firebrand, Also slightly shifted the arm hitbox inwards to the left from 5.5 ->5.2.
Edit (4): Increase sourspot hitbox by 2.5 -> 3.0, and shifted it outwards from 0 to 0.5
Edit (5): Changes:
- Strong hit frame lowered from 3 -> 2
- Overall attacking frames is now 7
- Sour spot BKG lowered from 50 -> 45
- Lingering hitbox BKG lowered from 50 -> 40

Before:

![Screenshot 2023-02-03 155807](https://user-images.githubusercontent.com/115908645/216711677-733d516c-5116-4e19-97c4-31a20abd5fd3.png)
![Screenshot 2023-02-03 155839](https://user-images.githubusercontent.com/115908645/216711690-6bc47e98-cf9f-4b15-9e42-acf45981bccb.png)

After:

![Screenshot 2023-02-03 175430](https://user-images.githubusercontent.com/115908645/216726502-f4c1cb0f-af3e-40a8-8288-3bd502f2ceaf.png)

![Screenshot 2023-02-03 175556](https://user-images.githubusercontent.com/115908645/216726528-95f44c5b-5443-4385-b124-a4b702e9f4e7.png)

Nakedkoops committed:
A bunch of changes to Mario's Fire Brand

Changes:

[=] Given a weaker lingering hitboxes thats been given the flowing changes:
    - Damage 8
    - KBG 100
[-] Strong hitboxes last from 5 frames to 3 frames
[=] The lingering hitboxes last for 7 frames
[+] The arm hitbox has been shifted to the right from 5 to 5.5
[=] shoulder hitbox size has been reduced from 3.5 to 2.5
[-] arm hitbox has reduced size from 6.5 to 5.0
[-] strong shoulder hitbox is also slightly weaker to differentiate it from the strong arm hitbox, changes are:
    -Damage 10
    - KGB 110

Nakedkoops committed:
Update specials.rs

Edit:

Changed arm hitbox size from 5 -> 5.5

Nakedkoops committed:
Update specials.rs

Nakedkoops committed:
Visual effects changes.

Changed the visuals of mario's firebrand slightly.

Nakedkoops committed:
Final touch ups to firebrand

 Increased shoulder (sour spot) hitbox from 2.5 to 3.0
Shifted sour spot hitbox outwards to the x axis from 0.0 to 0.5

Nakedkoops committed:
Update specials.rs

Riah321 commented:
Why are the effects so insanely bright and large? Why does Mario flash puke yellow? Please fix this!
Kills too early. Reduce BKB probably. With how easily mario will confirm this it's too strong.

Riah321 commented:
![2023020801131200-0E7DF678130F4F0FA2C88AE72B47AFDF](https://user-images.githubusercontent.com/82011099/217492804-0d170a38-b90f-4c4e-8b93-5eff58189d62.jpg)
![2023020801132800-0E7DF678130F4F0FA2C88AE72B47AFDF](https://user-images.githubusercontent.com/82011099/217492810-5959f7aa-7eb2-49e6-b9cb-06804919187e.jpg)

Nakedkoops committed:
Actual final changes

Changes:
Strong hit frame lowered from 3 -> 2
Overall attacking frames is now 7
Sour spot BKG lowered from 50 -> 45
Lingering hitbox BKG lowered from 50 -> 40

SuddyN reviewed (approved):
Code review looks good.
